### PR TITLE
fix: allow using existing session token(String) in place autocomplete

### LINKED ISSUE #689: Allow using existing session token(String) in place autocomplete
In our case the session token is not generated on the service that does the Google Places API call. Instead an existing token is passed from the client to facilitate correct billing. The token is not necessarily a UUID and there is no reason why only UUID should be allowed as implemented here #478. The API accepts any string as a token.

My request is to add another constructor to `SessionToken`:
```java
public SessionToken(String token) {
  this.token = token;
}
```

### PR DESCRIPTION
**Pull request by SethMyers**
Fixes #689

### CODE REVIEW

**Comment by SethMyers** [comment]
@googlebot I signed it!

File: src/main/java/com/google/maps/PlaceAutocompleteRequest.java [hunk 1]
```diff
@@ -67,6 +67,15 @@ public SessionToken(UUID uuid) {
       this.uuid = uuid;
     }
 
+    /**
+     * Construct a session that is a continuation of a previous session.
+     *
+     * @param token The unique token to be used as the identifier for this session.
```
Comment (arriolac) [inline src/main/java/com/google/maps/PlaceAutocompleteRequest.java:73]: I would expect `token` here to be a string representation of a UUID after reading this comment, but that's not the case given the implementation. While this provides convenience, I think this is potentially error-prone. I suggest either clarifying the comment, or keeping the existing API of constructing a `SessionToken` from a `UUID`. The latter seems like a clearer API to me, but if a convenience initializer is useful for others, I'm happy to merge in.

**Review by arriolac — CHANGES_REQUESTED** [review]

File: src/main/java/com/google/maps/PlaceAutocompleteRequest.java (hunk 1 above)
Comment (SethMyers) [reply src/main/java/com/google/maps/PlaceAutocompleteRequest.java:73]: would
`@param token The character string to be used as the unique identifier for this session.`
be clear enough?

Comment (vojtapol) [reply src/main/java/com/google/maps/PlaceAutocompleteRequest.java]: > keeping the existing API
The existing API does not work well if the session identifier is not a UUID.

**Review by arriolac — APPROVED** [review]

**Pull request merged**